.github/workflows/meson.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           path: build/meson-logs/meson-log.txt
 
   code-coverage:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - name: install libraries
         run: sudo apt-get install libjson-c-dev lcov
```

Comment:
we can try using this in the BSFishy/meson-build https://github.com/marketplace/actions/meson-build#gcovr-version
```
gcovr-version: 12.1.0
```